flink-sql-runner-dist/pom.xml

```
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.github.streamshub</groupId>
+        <artifactId>flink-sql</artifactId>
+        <version>0.0.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>flink-sql-runner-dist</artifactId>
+
+    <properties>
+        <maven.compiler.release>11</maven.compiler.release>
```

```suggestion
        <maven.compiler.release>17</maven.compiler.release>
```
?

Might be worth making the release version a property of its own.

---

yeap, I think the compiler config should move up into the parent. There's a separate issue to go to 17 too #3